<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerParticlesDisplayModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerParticlesDisplayModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Input Model</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="InputModelComboBox" native="true">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>27</height>
         </size>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="outputLabel">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Output Particles Model </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="OutputModelComboBox" native="true">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>27</height>
         </size>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="RegionComboBox">
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
       </widget>
      </item>

      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="TypeComboBox"/>
      </item>

      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color By</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="ColorComboBox"/>
      </item>

      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Glyph Type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="GlyphTypeComboBox"/>
      </item>

      <item row="7" column="0">
       <widget class="QLabel" name="scaleLabel">
        <property name="text">
         <string>Scale </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">y
       <widget class="QSlider" name="ParticlesScaleSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerParticlesDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputModelComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerParticlesDisplayModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>OutputModelComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
